data/json/effects.json

```diff
@@ -2073,6 +2073,52 @@
     "rating": "bad",
     "base_mods": { "vomit_chance": [ 500 ] }
   },
+  {
+    "type": "effect_type",
+    "id": "bile_stink",
+    "name": [ "Bile Stink" ],
+    "desc": [ "You're splattered with a greasy layer of liquified putrescence." ],
+    "apply_message": "You're covered in a foul smelling fluid!",
+    "rating": "bad",
+    "resist_traits": [ "FLOWERS", "ROSEBUDS", "SAPROVORE", "EATDEAD" ],
+    "base_mods": { "vomit_chance": [ 500, 2000 ] },
+    "enchantments": [
+      {
+        "values": [
+          { "value": "SOCIAL_PERSUADE", "add": -12 },
+          { "value": "SOCIAL_LIE", "add": -5 },
+          { "value": "SOCIAL_INTIMIDATE", "add": 10 },
```

Comment:
I don't buy the rationale, you stink, that doesn't make you more intimidating.
```suggestion
```

Reply:
I based it on the `skunk_spray` giving a +2 to intimidation, but it would actually make more sense on `bile_irritant`.

Reply:
Aha, well I'll be sure to get rid of that one too.